src/main/java/frc/team2412/robot/subsystems/ClimbSubsystem.java

```diff
@@ -1,11 +1,88 @@
 package frc.team2412.robot.subsystems;
 
+import com.ctre.phoenix.motorcontrol.TalonFXControlMode;
 import com.ctre.phoenix.motorcontrol.can.TalonFX;
 import edu.wpi.first.wpilibj.DoubleSolenoid;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 
 public class ClimbSubsystem extends SubsystemBase {
-    public ClimbSubsystem(TalonFX climbFixed1, TalonFX climbFixed2, TalonFX climbAngled1, TalonFX climbAngled2, DoubleSolenoid climbAngle) {
 
+    public static class ClimbConstants {
+        public static final double MAX_SPEED = 1;
+        public static final double STOP_SPEED = 0;
     }
-}
+
+    /**
+     * State of the cliimb subsystem
+     */
+    enum ClimbSubsystemState {
+        ENABLED, DISABLED
+    }
+
+    /**
+     * State of the solenoid to change the climb angle
+     */
+    enum HookArmState {
+        EXTENDED, UPRIGHT
+    }
```

Comment:
These should be under ClimbConstants

Reply:
From what we've done, my understanding is this: these are options for the enum, therefore constant values that do not change. The actual instantiated ClimbSubsystemState is a variable and should be in the class.